.github/docker/cluster_test_12/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # DO NOT MODIFY: THIS FILE IS GENERATED USING "make generate_ci_workflows"
 
-ARG bootstrap_version=5
+ARG bootstrap_version=5.1
```

Comment:
on the main branch, it is appropriate to bump this number up to 6. We just can't do that on release branches.

Reply:
Makes sense!